### PR TITLE
feat(js) Implement `Number.new`, `Date.now` and `Date.UTC` bindings

### PR DESCRIPTION
**Pull request by Hywan**
* `Date.now` is a static method,
* `Date.UTC` is a static method,
* To test `Date.UTC`, I needed `Number.new` so I created it,
* `Date.UTC` has 5 optional parameters (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/UTC) I didn't implement (cf. https://github.com/rustwasm/wasm-bindgen/issues/275#issuecomment-398549004). Is it OK?

### CODE REVIEW

File: src/js.rs [hunk 1]
```diff
@@ -431,6 +431,13 @@ extern {
     #[wasm_bindgen(constructor)]
     pub fn new() -> Date;
 
+    /// The `Date.now()` method returns the number of milliseconds
+    /// elapsed since January 1, 1970 00:00:00 UTC.
+    ///
+    /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now
+    #[wasm_bindgen(static_method_of = Date)]
+    pub fn now() -> Number;
```
Comment (fitzgen) [inline src/js.rs]: I think this should return a `u64` -- it isn't returning a `Number` *object* (as opposed to a number) so this isn't correct as-written.

File: src/js.rs [hunk 2]
```diff
@@ -516,6 +516,15 @@ extern {
...
     pub fn to_utc_string(this: &Date) -> JsString;
 
+    /// The `Date.UTC()` method accepts the same parameters as the
+    /// longest form of the constructor, and returns the number of
+    /// milliseconds in a `Date` object since January 1, 1970,
+    /// 00:00:00, universal time.
+    ///
+    /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/UTC
+    #[wasm_bindgen(static_method_of = Date, js_name = UTC)]
+    pub fn utc(year: Number, month: Number) -> Number;
```
Comment (fitzgen) [inline src/js.rs]: Ditto to every `Number` here.

**Comment by Hywan** [comment]
Anyway, the `Number::new` constructor is still in the patch even if it's not really related now ;-).

**Comment by fitzgen** [comment]
And `u32` isn't good enough in this case because we really do need numbers larger than 2^32 - 1 for date things, right? Ah well, I guess it has to be `f64`.

**Pull request merged**